Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Patch and build Olena from Git, then
 # Install OCR-D wrapper for binarization
-FROM ocrd/core
+FROM ocrd/core:v2.62.0 AS base
```

Comment:
If similar changes are made for more processors, new releases of OCR-D would require a lot of additional work.

Reply:
In principle – yes. But practically, the opposite is true – without pinning to a specific version, the module maintainers are always forced to react timely to things broken by upstream OCR-D. By pinning, we the maintainer can make a conscious decision on whether and when to update.

Of course, we should at some point do the same thing with pure Python requirements.  But for native installation (or ocrd_all via Docker) we currently cannot, because as long as our packages will live in a large unisolated venv, pinning would immediately shrink the intersection of compatible versions to zero. On the other hand, Docker modules are the way to go for greater isolation, and pinning here will now on offer some protection against breaking changes.

Reply:
Yeah, I think the added effort of hardcoding a specific version is worth the reproducibility and since it does not concern the deployment with ocrd_all (yet), I see no reason not to.